core/com.b2international.snowowl.datastore.server.tests/src/com/b2international/snowowl/core/validation/ValidationThreadPoolTest.java

```diff
@@ -54,13 +67,12 @@ public void testConcurrentExpensiveJobs() {
 		}
 
 		Promise.all(validationPromises).getSync();
+		System.out.println();
```

Comment:
Remove `println` calls. If you would like to separate the log output of the test cases then use a `JUnit Rule` like this class: `com.b2international.snowowl.test.commons.TestMethodNameRule`.